database/queries/add_career.js

```diff
@@ -0,0 +1,45 @@
+const dbConnection = require('../db_connection');
+const newCareer = `INSERT INTO careers (
```

Comment:
Could we call this newCareerQuery? And wouldn't it be nicer to just be part of the function below?